src/main/java/io/javalin/translator/json/Jackson.kt

```diff
@@ -6,24 +6,38 @@
 
 package io.javalin.translator.json
 
+import com.fasterxml.jackson.databind.Module
 import com.fasterxml.jackson.databind.ObjectMapper
 
 object JavalinJacksonPlugin {
 
     private var objectMapper: ObjectMapper? = null
 
+    private fun createObjectMapper(): ObjectMapper {
```

Comment:
Can be simplified by turning it into an expression function and by using the catch: 

```kotlin
private fun createObjectMapper() = try {
    val className = "com.fasterxml.jackson.module.kotlin.KotlinModule"
    ObjectMapper().registerModule(Class.forName(className).getConstructor().newInstance() as Module)
} catch (e: Exception) {
    ObjectMapper()
}
```

Reply:
I've updated the implementation with your suggestion and tested the implementation locally - seems to work!